docs/integrations/sources/mongodb-v2.md

```diff
@@ -180,16 +156,43 @@ When Schema is not enforced there is not way to deselect fields as all fields ar
 
 ## Limitations & Troubleshooting
 
+### MongoDB Oplog and Change Streams
+
+[MongoDB's Change Streams](https://www.mongodb.com/docs/manual/changeStreams/) are based on the [Replica Set Oplog](https://www.mongodb.com/docs/manual/core/replica-set-oplog/). This has retention limitations.  Syncs that run less frequently than the retention period of the Oplog may encounter issues with missing data.
+
+We recommend adjusting the oplog size for your MongoDB cluster to ensure it holds at least 24 hours of changes. For optimal results, we suggest expanding it to maintain a week's worth of data. To adjust your oplog size, see the corresponding tutorials for [MongoDB Atlas](https://www.mongodb.com/docs/atlas/cluster-additional-settings/#set-oplog-size) (fully-managed) and [MongoDB shell](https://www.mongodb.com/docs/manual/tutorial/change-oplog-size/) (self-hosted). This will require changes to both the minimum Oplog window in time as well as the minimum Oplog size in storage space.
```

Comment:
```suggestion
We recommend adjusting the Oplog size for your MongoDB cluster to ensure it holds at least 24 hours of changes. For optimal results, we suggest expanding it to maintain a week's worth of data. To adjust your Oplog size, see the corresponding tutorials for [MongoDB Atlas](https://www.mongodb.com/docs/atlas/cluster-additional-settings/#set-oplog-size) (fully-managed) and [MongoDB shell](https://www.mongodb.com/docs/manual/tutorial/change-oplog-size/) (self-hosted). This will require changes to both the minimum Oplog window in time as well as the minimum Oplog size in storage space.
```